scripts/karma-classic.coffee

```diff
@@ -0,0 +1,140 @@
+# Description:
```

Comment:
This comes from an npm module, which can be included instead of added directly:
https://github.com/hubotio/hubot/blob/master/docs/index.md#scripts